addons/website/tests/test_ui.py

```diff
@@ -292,7 +287,42 @@ def test_07_snippet_version(self):
                 </xpath>
             """,
         }])
-        self.start_tour(self.env['website'].get_client_action_url('/'), 'snippet_version', login='admin')
+        self.start_tour(self.env['website'].get_client_action_url('/'), 'snippet_version_1', login='admin')
```

Comment:
Why did you actually have to split the tour in two ?

Reply:
Previously, the tour dropped the snippets, then accessed the html of the SnippetsMenu to increase the version of a snippet template, then clicked on the snippets to see if the "outdated" warning showed up.

This is not possible anymore since snippets are defined in SnippetsMenu as objects and are no longer in the DOM. So I split the tour in two, and use python to increase the version instead of using JS.